app/controllers/core_data_connector/public/v1/public_controller.rb

```diff
@@ -11,7 +11,9 @@ def base_query
           return item_class.none unless params[:project_ids].present?
 
           if nested_resource?
-            item_class.joins(build_base_sql)
+            item_class
```

Comment:
The `INNER JOIN` included in `build_base_sql` will contain the `order` column. Can we prefix the `.order` column with a table name to avoid ambiguous column names?